transit_odp/timetables/loaders.py

```diff
@@ -181,6 +189,31 @@ def load_serviced_organisation(self):
                 )
             )
 
+    def load_operating_profiles(self, vehicle_journeys):
+        operating_profiles = self.transformed.operating_profiles
+        if not operating_profiles.empty and not vehicle_journeys.empty:
+            operating_profiles.reset_index(inplace=True)
+            vehicle_journeys = vehicle_journeys.rename(
+                columns={"service_code_x": "service_code"}
+            )
+            vehicle_journeys = vehicle_journeys[
```

Comment:
Is this required as the other columns are ignored when `operating_profiles_objs` are created